.github/workflows/build.yml

```diff
@@ -93,26 +93,6 @@ jobs:
         with:
           ref: 'dev'
           fetch-depth: 0
-
-      - name: Cache OpenShift CLI
```

Comment:
doesnt `make server-build` and `make server-deploy` need the OC CLI?

Reply:
OC CLI is preinstall in ubuntu latest ....